Capgemini.Xrm.CdsDataMigrator/Capgemini.Xrm.CdsDataMigratorLibrary/Presenters/IImportMappingsFormView.cs

```diff
@@ -0,0 +1,25 @@
+﻿
+using Capgemini.Xrm.CdsDataMigratorLibrary.Core;
+using Capgemini.Xrm.CdsDataMigratorLibrary.Models;
+using Capgemini.Xrm.DataMigration.Config;
+using Capgemini.Xrm.DataMigration.Model;
+using Microsoft.Xrm.Sdk;
+using System;
+using System.Collections.Generic;
+using System.Windows.Forms;
+
+namespace Capgemini.Xrm.CdsDataMigratorLibrary.Presenters
+{
+    public interface IImportMappingsFormView
+    {
+        CrmSchemaConfiguration SchemaConfiguration { get; }
+        event EventHandler OnVisible; 
+
+        IEnumerable<string> EntityList { get; set; }
+        List<DataGridViewRow> Mappings { get; set; }
+
+        void Close();
+        DialogResult ShowMessage(string message, string caption, MessageBoxButtons buttons, MessageBoxIcon icon);
```

Comment:
Could this be added to `CdsMigratorPluginControl` and `INotifier` as an 'info' message with no actions (apart from OK? `ShowInfoMessage` name could work with only `message` and `caption` parameters. `buttons` can be fixed to `OK` and `icon` to `Info`.

This keeps all notification actions in one place.

Reply:
This might be a little complicated as it depends where the `ImportMappingsFormPresenter` is newed up.

Reply:
Leaving for now - to be discussed in next catch-up.